content/en/docs/reference/kubectl/conventions.md

```diff
@@ -19,6 +19,16 @@ For a stable output in a script:
 * Fully-qualify the version. For example, `jobs.v1.batch/myjob`. This will ensure that kubectl does not use its default version that can change over time.
 * Don't rely on context, preferences, or other implicit states.
 
+## Subresources
+
+* You can use the `--subresource` flag for kubectl commands like `get`, `patch`,
```

Comment:
Nit: maybe worth mentioning this is alpha flag.

Reply:
whoops, not sure how I missed that. Fixed :)